circle-mlir/circle-mlir/tools-test/onnx2circle-value-test/util_validation.py

```diff
@@ -0,0 +1,46 @@
+import numpy as np
+import json
+
+
+# Get PEIR(Peak Error-to-Interval Ratio) from two outputs
+# Each output is an array, and it returns PEIR as string type.
+def get_peir_from(baseline_output, target_output):
+    if baseline_output.dtype == 'bool':
+        diff = np.absolute(baseline_output ^ target_output).reshape(-1)
+    else:
+        diff = np.absolute(baseline_output - target_output).reshape(-1)
+
+    baseline_min = np.min(baseline_output.reshape(-1))
+    baseline_max = np.max(baseline_output.reshape(-1))
+
+    PEAK_ERROR = np.max(diff)
+    if baseline_max.dtype == 'bool':
+        # ignore PEIR for bool type
+        PEIR = 0
+    else:
+        INTERVAL = baseline_max - baseline_min
+        PEIR = PEAK_ERROR / INTERVAL
```

Comment:
(Optional)
If all values in `baseline_output` are identical, the interval would be `0`. This would cause a division by zero when computing `PEIR = PEAK_ERROR / INTERVAL`. Adding a check to handle or bypass this case would improve the robustness of the code.

Reply:
ditto